### PR TITLE
fep-820 font weight

### PR DESCRIPTION
**Pull request by ashB100**
# Description

The new [design](https://www.figma.com/file/Rn2XQNSA3vuIwXAtbRLjhT/Foundations?node-id=541%3A2064&t=YNyDueL6mK0FUo5F-0.) introduces a Roboto font-weight of 700. 

This PR adds Roboto Bold (font-weight 700) font face. This is a breaking change including the following changes for content using font-family Roboto:
* Rename --font-weight-bold to --font-weight-medium 
* Rename modifier --strong to --medium
* Create --font-weight-bold using Roboto 700
* Create modifier --strong using --font-weight-bold
* Change the style for strong and b HTML elements to use --font-weight-bold

The Canopy documentation is updated to reflect the new styles.  

### To help with testing:

Here are the different font weights:
  --font-weight-thin: 100;
  --font-weight-light: 300;
  --font-weight-regular: 400;
  --font-weight-medium: 500;
  --font-weight-bold: 700;

Default weights for the different font sizes:
7, 6, 5, 4, 3, 2: --font-weight-light
1, 0.8, 0.6: --font-weight-regular

[Link to design](https://www.figma.com/file/Rn2XQNSA3vuIwXAtbRLjhT/Foundations?node-id=541%3A2064&t=YNyDueL6mK0FUo5F-0.)

# Checklist:

- [x] The commit messages follow the [convention for this project](./blob/master/docs/CONTRIBUTING.md#conventional-commits)
- [ ] I have provided an adequate amount of test coverage
- [ ] I have added the functionality to the [test app](./blob/master/docs/CONTRIBUTING.md#build-test-application)
- [x] I have provided a story in storybook to document the changes
- [ ] I have added the documentation
- [ ] I have added any new public feature modules to public-api.ts

### CODE REVIEW

File: .storybook/preview-head.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
 
   .sbdocs-h4 {
     font-size: var(--text-fs-2-size) !important;
-    font-weight: var(--font-weight-bold) !important;
+    font-weight: var(--font-weight-medium) !important;
```
Comment (Xinchro) [inline .storybook/preview-head.html:37]: is this the only one that should change to bold? i see another `bold` below here

**Review by paul-request — DISMISSED** [review]
Look good, nice work @ashB100

File: .storybook/preview-head.html (hunk 1 above)
Comment (ashB100) [reply .storybook/preview-head.html:37]: Thanks for pointing it out Walter.

**Review by Xinchro — APPROVED** [review]
🎉

**Review by Xinchro — APPROVED** [review]

**Review by Xinchro — APPROVED** [review]

**Review by paul-request — APPROVED** [review]

**Pull request merged**